nix/docker.nix

```diff
@@ -23,6 +23,10 @@ in pkgs.dockerTools.buildImage {
     architecture = "amd64";
     os = "linux";
 
+    Env = [
```

Comment:
Adds the SSL certificates from the ca-bundle to the environment so that our binaries can pick them up for http requests